src/FSharpVSPowerTools.Core/Utils.fs

```diff
@@ -29,9 +29,15 @@ module Array =
 
     /// Returns true if one array has trailing elements equal to another's.
     let endsWith (suffix: _ []) (whole: _ []) =
-        whole 
-        |> Array.rev
-        |> startsWith (Array.rev suffix)
+        let rec loop step =
+            if step > suffix.Length then true
+            elif step > whole.Length then false
+            elif suffix.[suffix.Length - step] = whole.[whole.Length - step] then loop (step + 1)
```

Comment:
It's small thing but you can hoist `suffix.Length` and `whole.Length` to values outside the `loop` function.